Shaders/KopernicusShaders/Assets/Shaders/ExportAssetBundle.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEditor;
+
+public class CreateAssetBundles
+{
+	[MenuItem ("Assets/Build AssetBundles")]
+	static void BuildAllAssetBundles ()
+	{
+		// Put the bundles in a folder called "AssetBundles"
+		var outDir = "D:/gh/Steam/steamapps/common/Kerbal Space Program/GameData/Kopernicus/Shaders";
```

Comment:
Maybe use a relative path? Not blocking - I can do that myself, I just dont like hardcoded paths :D